apps/remote_control/test/lexical/remote_control/build/error_test.exs

```diff
@@ -9,28 +9,79 @@ defmodule Lexical.RemoteControl.Build.ErrorTest do
   def compile(source) do
     doc = Document.new("file:///unknown.ex", source, 0)
 
-    case Code.string_to_quoted(source) do
+    case Code.string_to_quoted(source, columns: true) do
       {:ok, quoted_ast} ->
+        if System.version() >= "1.15" do
+          do_compile_after_1_15(doc, quoted_ast)
+        else
+          do_compile(doc, quoted_ast)
+        end
+
+      error ->
+        Logger.warning("error is #{inspect(error)}")
+        {doc, error}
+    end
+  end
+
+  defp do_compile(doc, quoted_ast) do
+    try do
+      modules = for {m, _b} <- Code.compile_quoted(quoted_ast, "/unknown.ex"), do: m
```

Comment:
this is an instance of the "short" form of a comprehension making code that's less readable.
```suggestion
      modules = for {module _bytecode} <- Code.compile_quoted(quoted_ast, "/unknown.ex") do 
        module 
      end      
```